Access to non-existent attribute in `Prob`

The following line accesses the `description` attribute which doesn't exist in a `Probe` object.

https://github.com/con/nwb2bids/blob/f0fb7253c949cd471cb2b73c861d111085c9fffc/src/nwb2bids/bids_models/_probes.py#L48

This error is detected by mypy as the following.

```console
src/nwb2bids/bids_models/_probes.py:48: error: "Probe" has no attribute "description"  [attr-defined]
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -177,10 +177,6 @@ ignore_missing_imports = true
 
 # === Temporary mypy ignores for selected files ===
 [[tool.mypy.overrides]]
-module = ["*._probes"]
-disable_error_code = ["attr-defined"]
-
-[[tool.mypy.overrides]]
 module = ["nwb2bids.bids_models._bids_session_metadata"]
 disable_error_code = ["arg-type"]
 

--- a/src/nwb2bids/bids_models/_probes.py
+++ b/src/nwb2bids/bids_models/_probes.py
@@ -169,7 +169,7 @@ class ProbeTable(BaseMetadataContainerModel):
     def _check_fields(self) -> None:
         self._internal_notifications = []
 
-        probes_missing_description = [probe for probe in self.probes if probe.description is None]
+        probes_missing_description = [probe for probe in self.probes if getattr(probe, "description", None) is None]
         for _ in probes_missing_description:
             notification = Notification.from_definition(
                 identifier="MissingDescription",

